src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -18,389 +18,381 @@
+
+			writer.AppendLineInvariant("using {0};", _defaultNamespace);
+
+			// For Subclass build functionality
+			writer.AppendLineInvariant("");
+			writer.AppendLineInvariant("#if __ANDROID__");
+			writer.AppendLineInvariant("using _View = Android.Views.View;");
+			writer.AppendLineInvariant("#elif __IOS__");
+			writer.AppendLineInvariant("using _View = UIKit.UIView;");
+			writer.AppendLineInvariant("#elif __MACOS__");
+			writer.AppendLineInvariant("using _View = AppKit.NSView;");
+			writer.AppendLineInvariant("#elif __WASM__");
+			writer.AppendLineInvariant("using _View = Windows.UI.Xaml.UIElement;");
+			writer.AppendLineInvariant("#elif NET461");
+			writer.AppendLineInvariant("using _View = Windows.UI.Xaml.UIElement;");
+			writer.AppendLineInvariant("#endif");
+
+			writer.AppendLineInvariant("");
+
+			var topLevelControl = _fileDefinition.Objects.First();
+
+			if (topLevelControl.Type.Name == "ResourceDictionary")
+			{
+				BuildEmptyBackingClass(writer, topLevelControl);
+
+				BuildTopLevelResourceDictionary(writer, topLevelControl);
+			}
+			else
+			{
+				_className = GetClassName(topLevelControl);
+
+				using (writer.BlockInvariant("namespace {0}", _className.ns))
+				{
+					AnalyzerSuppressionsGenerator.Generate(writer, _analyzerSuppressions);
+
+					var controlBaseType = GetType(topLevelControl.Type);
+
+					using (writer.BlockInvariant("public sealed partial class {0} : {1}", _className.className, controlBaseType.ToDisplayString()))
+					{
+						var isDirectUserControlChild = IsUserControl(topLevelControl.Type, checkInheritance: false);
+
+						using (Scope("{0}{1}".InvariantCultureFormat(_className.ns.Replace(".", ""), _className.className)))
+						{
+							using (writer.BlockInvariant(BuildControlInitializerDeclaration(_className.className, topLevelControl)))
+							{
+								if (IsApplication(topLevelControl.Type))
+								{
+									BuildApplicationInitializerBody(writer, topLevelControl);
+								}
+								else
+								{
+									BuildGenericControlInitializerBody(writer, topLevelControl, isDirectUserControlChild);
+									BuildNamedResources(writer, _namedResources);
+								}
+
+								if (isDirectUserControlChild)
+								{
+									writer.AppendLineInvariant("return content;");
+								}
+							}
+
+							if (isDirectUserControlChild)
+							{
+								using (writer.BlockInvariant("public {0}(bool skipsInitializeComponents)", _className.className))
+								{
+								}
+
+								using (writer.BlockInvariant("private void InitializeComponent()"))
+								{
+									writer.AppendLineInvariant("Content = (_View)GetContent();");
+								}
+							}
+
+							BuildPartials(writer, isStatic: false);
+
+							BuildBackingFields(writer);
+
+							BuildChildSubclasses(writer);
+
+						}
+
+						BuildInitializeXamlOwner(writer);
+					}
+				}
+			}
+
+			BuildXamlApplyBlocks(writer);
+
+			// var formattedCode = ReformatCode(writer.ToString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305744)